department-of-veterans-affairs/caseflow
# Remove filed by VA.gov HLR feature toggle

Resolves N/A

### Description
Remove Filed by VA.gov HLRH feature toggle

### Acceptance Criteria

### Testing Plan
1. Go to intake
2. Select HLR form
3. Search for Veteran
4. Verify option to select if filed by VA.gov is still visible
